qa-tests-backend/src/test/groovy/CloudSourcesTest.groovy

```diff
@@ -29,7 +29,7 @@ class CloudSourcesTest extends BaseSpecification {
         "verify we have discovered clusters"
         // The initial sync may take a bit since we are connecting to the "Red Hat1" OCM organization which hosts
         // a lot of clusters.
-        withRetry(10, 10) {
+        withRetry(10, 30) {
```

Comment:
Let's increase the number of retries instead of the pause seconds.

Reply:
I didn't do that because the retries spam the test logs. Why do you prefer many retries?

Reply:
We've tried to optimize every test to use more retries instead of waiting longer to lower the time each test takes. so let's align with this here as well.